### ISSUE TITLE
Request: Donation options

### ISSUE THREAD

**Issue by Randelung** [jakearchibald/svgomg#200]
Hello!

Your web app has contributed greatly to my efficiency and I would like to show my appreciation.
Do you have a way to donate a few bucks? (PayPal doesn't count :))

Cheers

**Comment by venerated** [comment]
Bump for this. Surprised there's no option to even donate a coffee or something to the contributors!

**Comment by jakearchibald** [comment]
I don't think this web app needs donations. It's fairly low maintenance, and I can sometimes get away with going it in work hours.

However, the underlying library accepts donations: https://opencollective.com/svgo

It's probably worth adding a link to that on the site.
